CourseGrab/ViewControllers/Home/HomeTableViewCell.swift

```diff
@@ -161,6 +183,15 @@ class HomeTableViewCell: UITableViewCell {
             }
         }
     }
+    func changeBackground(isHighlighted: Bool){
+        if isHighlighted{
+            self.containerView.backgroundColor = UIColor(red: 220/255, green: 220/255, blue: 220/255, alpha: 1)
+        }
+        else{
+         self.containerView.backgroundColor = .white
```

Comment:
You don't need self and make the else inline. Also think about color literals : )